homeassistant/components/camera/synology.py

```diff
@@ -194,6 +214,7 @@ def __init__(self, hass, websession, config, camera_id,
         self._camera_path = camera_path
         self._auth_path = auth_path
         self._timeout = timeout
+        self._is_recording = is_recording
```

Comment:
I don't see how this value will be changed, so whatever the value is on startup will be the value always?

Reply:
Yes it’ll only update once during startup but at least it’ll show correct state instead of always idle

I’m preparing another PR to update states periodically in background